Resources/Prototypes/Nyanotrasen/Entities/Body/Prototypes/felinid.yml

```diff
@@ -4,14 +4,14 @@
   root: torso
   slots:
     head:
-      part: HeadHuman
```

Comment:
stuff in Nyanotrasen doesn't need DeltaV comments

Reply:
Removed them.